gap/tools/io.gi

```diff
@@ -313,3 +313,212 @@ function(str)
 
   return IteratorByFunctions(record);
 end);
+
+InstallGlobalFunction(ReadMultiplicationTable,
+function(arg)
+  local name, line_nr, file, ReadMultiplicationTableLine, i, line, n, lines;
+  if Length(arg) = 1 then
+    name    := arg[1];
+    line_nr := 0;
+  elif Length(arg) = 2 then
+    name    := arg[1];
+    line_nr := arg[2];
+  else
+    ErrorNoReturn("Semigroups: ReadMultiplicationTable: usage,\n",
+                  "there should be at most 2 arguments,");
```

Comment:
"there should be 1 or 2 arguments" since this error will be shown if there are no arguments.